src/components/Shared/DataGrid/index.js

```diff
@@ -32,7 +32,7 @@ export function DataGrid({
 
     let updatedRow = { ...newRow }
 
-    if (column?.onChange)
```

Comment:
why do we have a change in the datagrid in this commit ? it's not related to it.